testing.Makefile

```diff
@@ -76,10 +72,44 @@ testing/marketplace/deployer/envsubst: \
 		.testing/marketplace/deployer/envsubst/standard
 
 
-.PHONY: testing/all
-testing/all: \
+.PHONY: testing/integration
```

Comment:
I prefer `test/integration` and `test/py`, or `tests/integration`. `testing` makes sense in this context since that's the name of the Makefile. Would recommend consider `tests`--and renaming accordingly.

Reply:
Renamed to `tests/*`.